src/Server.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive-swoole for the canonical source repository
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive-swoole/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Zend\Expressive\Swoole;
+
+use Swoole\Http\Server as SwooleHttpServer;
+use Zend\Expressive\Swoole\Exception\InvalidArgumentException;
+
+use function in_array;
+use function array_replace;
+
+class Server
```

Comment:
Note to self: this is really a `ServerFactory`, and thus the `ServerFactory` is a `ServerFactoryFactory`. (Horrible naming, but correct.)